src/wrapper/Plumed.h

```diff
@@ -3405,7 +3405,7 @@ __PLUMED_IMPLEMENT_FORTRAN(plumed_f_use_count,PLUMED_F_USE_COUNT,(char*c,int*i),
 /* note: flags & (~0x1ffffff) removes bits that are set here (code and size) */
 
 #define __PLUMED_IMPLEMENT_F_SAFEPTR_INNER(type,type_,size,code,suffix) \
-plumed_safeptr plumed_f_safeptr_ ## type_ ## suffix(type*val,__PLUMED_WRAPPER_STD size_t nelem,__PLUMED_WRAPPER_STD size_t*shape,__PLUMED_WRAPPER_STD size_t flags,void*opt) {\
+plumed_safeptr plumed_f_safeptr_ ## type_ ## suffix(void*val,__PLUMED_WRAPPER_STD size_t nelem,__PLUMED_WRAPPER_STD size_t*shape,__PLUMED_WRAPPER_STD size_t flags,void*opt) {\
```

Comment:
I guess this change has no effect (the function is only called by Fortran), but it's ok. I also guess it is not a problem to continue defining it as with
```
${t}$(kind=${k}$)                     :: val
```
in the plumed.f90 (old) interface.

Reply:
Yes, the difference is only "syntactic sugar". The old Fortran interfaces should also work without problems.